main.go

```diff
@@ -84,8 +84,7 @@ func initFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&leaderElectionRetryPeriod, "leader-elect-retry-period", 2*time.Second,
 		"Duration the LeaderElector clients should wait between tries of actions (duration string)")
 
-	fs.StringVar(&watchFilterValue, "watch-filter", "",
-		fmt.Sprintf("Label value that the controller watches to reconcile cluster-api objects. Label key is always %s. If unspecified, the controller watches for all cluster-api objects.", clusterv1.WatchLabel))
+	fs.StringVar(&watchFilterValue, "watch-filter", "", fmt.Sprintf("Label value that the controller watches to reconcile cluster-api objects. Label key is always %s. If unspecified, the controller watches for all cluster-api objects.", clusterv1.WatchLabel)) //nolint:lll
```

Comment:
I wonder if something like https://github.com/golangci/golangci-lint/issues/207#issuecomment-534902640 would work out for us too?

Reply:
seems like it only works for comments, ignoring a line of code breaks file validation